packages/logger/__tests__/unit/loggerFactory.spec.ts

```diff
@@ -0,0 +1,21 @@
+import { LoggerFactory } from "../../src/LoggerFactory";
+
+describe("LoggerFactory try to create type '$type'", () => {
+  // We can add more in the future
+  it.each([
+    {
+      type: "consola-gcp",
+      expected: "Object",
```

Comment:
It doesn't really check if consola-gcp type has been created

Reply:
I know, since i changed it from the class => function factory I do not know how to do it without some kind of metadata on the object which will be there only for this test.

Reply:
Remove it then